wait/member.go

```diff
@@ -901,16 +903,20 @@ func (a *MemberAwaitility) waitForResource(namespace, name string, object runtim
 	err := wait.Poll(a.RetryInterval, a.Timeout, func() (done bool, err error) {
 		if err := a.Client.Get(context.TODO(), test.NamespacedName(namespace, name), object); err != nil {
 			if errors.IsNotFound(err) {
+				a.T.Logf("resource '%s' in namesapace '%s' not found", name, namespace)
 				return false, nil
 			}
+			a.T.Logf("unexpected error when looking for resource '%s' in namesapace '%s'", name, namespace)
 			return false, err
 		}
+		a.T.Logf("resource '%s' in namesapace '%s' found", name, namespace)
 		return true, nil
 	})
 	require.NoError(a.T, err)
 }
 
 func (a *MemberAwaitility) waitForService() {
```

Comment:
Btw I think we should rename all these web hook specific waits to `waitForWebHookService()`, `waitForWebHookDeployment()`, etc instead of `waitForDeployment()` to avoid confusion since it's not the only deployment which is running in the member.